contrib/polybar.aur/PKGBUILD

```diff
@@ -1,8 +1,8 @@
 # Maintainer: Michael Carlberg <c@rlberg.se>
```

Comment:
Should we update that or add a new contributor?

Reply:
I updated it. Since I am the only one that updates that package, I should also be the contact person.